src/transformers/pipelines/feature_extraction.py

```diff
@@ -40,19 +40,17 @@ class FeatureExtractionPipeline(Pipeline):
             the associated CUDA device id.
     """
 
-    def _sanitize_parameters(self, truncation=None, **kwargs):
-        preprocess_params = {}
-        if truncation is not None:
-            preprocess_params["truncation"] = truncation
+    def _sanitize_parameters(self, tokenize_kwargs=None, **kwargs):
```

Comment:
Unfortunately we have to keep `truncation` in there since it was already allowed in order to NOT break anything.

Something like:

```python
def _sanitize_parameters(self, truncation=None, tokenize_kwargs=None, **kwargs):
    # handle tokenize_kwargs first
    if truncation is not None:
         if 'truncation' in tokenize_kwargs:
             raise ValueError("This is defined twice")
         tokenize_kwargs["truncation"] = truncation

Reply:
But the problem is truncation is already a parameter of the tokenizer, why the parameter should be kept separately?

Reply:
Because it was there before, hence users might (and probably HAVE) started using it.
And since we cannot break user code, we have to keep backward compatibility with it.